### PR TITLE
Implement redsigned donate page 3983

### LINKED ISSUE #3983: Implement Redesigned Donate Page (mobile and desktop)
### Overview

As developers, we need to implement the new Figma design for the Donate Page, in both mobile and desktop views. 

<details>
<summary>Existing Design</summary>

Donate page current mobile

Screenshot_20230215_015012

</details>

<details>
<summary>New Design (with update: see first Action Item)</summary>

donate develop this version red star

</details>

### Action Items

- [x] Implement Figma design for mobile view, with the following update: The blue color in the donate form should instead match the dark blue color in https://www.hackforla.org/assets/images/about/hfla-donate.gif
- [x] Consider the existing mobile and desktop pages, and from the new mobile design, extrapolate a corresponding desktop design and implementation.  
- [x] This issue will require changes to `_includes/donate-card.html`,  which is included in both `pages/donate.html` and `pages/join-us.html`.  To prevent changes to the Join Us page, make a copy of the unmodified `_includes/donate-card.html`, rename it `_includes/join-donate-card.html,` and update `pages/join-us.html` to include `_includes/join-donate-card.html`
- [x] Check to ensure that the appearance of the Join Us page is unchanged, except for  the updated donation circle graphic.  
- [x] Submit a pull request with the proposed changes

### Resources/Instructions
[Donate page](https://hackforla.org/donate)
[pages/donate.html](https://github.com/hackforla/website/blob/9a7168d42d43d5de8cc6e3995402bd8746e6b17a/pages/donate.html)
[_includes/donate-card.html](https://github.com/hackforla/website/blob/gh-pages/_includes/donate-card.html)
[Join Us page](https://hackforla.org/join)
[pages/join-us.html](https://github.com/hackforla/website/blob/gh-pages/pages/join-us.html)
[Figma design - marked with red star](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=24579%3A285100&t=C14tX0fL6J4V5RWB-4)
[_sass/components/_donate.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/components/_donate.scss)

### PR DESCRIPTION
**Pull request by Thinking-Panda**
Fixes #3983

### What changes did you make and why did you make them ?

  - A copy of donate-card.html was saved as join-donate-card.html and included in join-us.html. This was done so any changes to the Donate page does not affect the Join Us page.
  - In donate.html, the text content was changed based on Figma design and the solidarity-pana.svg was replaced with donation-graphic.svg. CSS class "donation-gif" was added so the image has a shadow.
  - In donate-card.html, text content was changed based on Figma design. New CSS class "page-card--donate", "donation-grid",  and "donation-body" was added for a responsive layout in tablet and desktop viewports.
  - In _donate.scss. img{} was removed as the class "join-us-doantion-img" was responsive with doantion-graphic.svg
  - I have added screenshots of different viewports below

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

donate_mobile
donate_tablet
donate_desktop

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
Redesigned_Donate_Mobile
Redesigned_Donate_Tablet
Redesigned_Donate_Desktop

</details>

### CODE REVIEW

**Comment by roslynwythe** [comment]
Availability: 3/19 2 - 4 pm
ETA EOD 3/19

**Comment by Suman2795** [comment]
Review ETA : 19/03/23
Availability :3 hours

**Comment by roslynwythe** [comment]
Thank you @Thinking-Panda for your excellent work on this issue.  The branches are setup correctly, the PR is descriptive, and the branch looks great in the browser at all screen sizes.  I was a bit surprised at how extensive the changes were in the .html and .scss files.  It would be useful for reviewers if you could explain a bit about those changes.

**Review by roslynwythe — DISMISSED** [review]
Thank you @Thinking-Panda for your excellent work on this issue. The branches are setup correctly, the PR is descriptive, the html and scss are clean and well organized, and the branch looks great in the browser at all screen sizes.

**Comment by Thinking-Panda** [comment]
Hello @roslynwythe and @Suman2795 , I have edited the PR with details of code changes in each file. Hope this helps.

**Comment by jdingeman** [comment]
Hello @Suman2795 - wanted to check in for your review?

@Thinking-Panda - it looks like there are merge conflicts. You may want to identify and resolve these before the merge can be made.

**Comment by roslynwythe** [comment]
Hi @Thinking-Panda it looks like in a recent PR #4210, line 14 in _includes/donate-card.html was modified to update the alt text.  So when resolving the merge conflict, you will want to retain that line.  Let me know if you need assistance with that.

**Review by Suman2795 — DISMISSED** [review]
Nice work @Thinking-Panda . The branches are merged correctly, website works good on the local machine in different modes and code changes are as per the action items in the issue.

**Comment by Thinking-Panda** [comment]
> Hi @Thinking-Panda it looks like in a recent PR #4210, line 14 in _includes/donate-card.html was modified to update the alt text. So when resolving the merge conflict, you will want to retain that line. Let me know if you need assistance with that.

Thank you @roslynwythe. I really need your assistance on resolving the conflict. I noticed the conflict but wasn't sure how to resolve it. Please let me know how to identify and retain that line.

**Review by roslynwythe — APPROVED** [review]
@Thinking-Panda thank you for resolving the merge conflict and then following up with the update to alt text in donate.html.  I've re-reviewed the issue and everything still checks out fine in the browser including the alt text.

**Comment by roslynwythe** [comment]
Hi @Suman2795 this is a reminder that we have to re-review the PR.  After resolving the merge conflict, a new commit was required to restore the alt text on the donation graphic.

**Review by Suman2795 — DISMISSED** [review]

**Review by roslynwythe — CHANGES_REQUESTED** [review]
@Thinking-Panda There is a possible problem with moving the images to donate.html: that is, this page: https://www.hackforla.org/join that also includes donate-card.html.   If you look at the Join Us page in your branch, you will see that it is missing the elements that were moved from donate-card.html to donate.html, so I need to ask that you restore those elements into donate-card.html.

**Comment by Thinking-Panda** [comment]
@roslynwythe , I see the missing image. I will make changes needed and update here.

**Comment by mademarc** [comment]
Review ETA:  3/23/2023
Availability: 7:05PM Monday

**Review by mademarc — DISMISSED** [review]
Hey  @Thinking-Panda the ul  elements looks very well updated and the media queries code was coded done correctly.

**Review by roslynwythe — APPROVED** [review]

**Review by roslynwythe — CHANGES_REQUESTED** [review]
Hi @Thinking-Panda please advise, are the changes requested in https://github.com/hackforla/website/issues/3983#issuecomment-1496503522 clear?

**Comment by Thinking-Panda** [comment]
Changes requested are being worked on. New commits will me made by 4/17/23

**Comment by Thinking-Panda** [comment]
@roslynwythe , @Suman2795 , @mademarc - Changes requested has been made after the issue was re-written. Please re-review this PR and let me know your thoughts.

**Review by roslynwythe — CHANGES_REQUESTED** [review]
Hi @Thinking-Panda there is a problem with overlapping text next to the animated donation gif from screen widths 480 px to 700 px.  This is at 600px width.  
image

File: _includes/donate-card.html [hunk 1]
```diff
@@ -1,46 +1,29 @@
...
-      products with Government and nonprofit partners, we are standardizing
-      our infrastructure and procedures. This enables Hack for LA to provide
-      world-class training to our members and valuable, scalable, impactful
-      products to the community.
-    </p>
-    <img class="join-us-donation-img" src="/assets/images/join-us/donation-graphic.svg" alt="Monthly: Hosting, Mailing List services, zoom; Annual: Domain Names; Occasional: Stickers, T-shirts" />
-    <h3 class="title6 page-card--large-icon-header-secondary">How to help us continue to do impactful work</h3>
-    <ul>
+<div class="page-card card-primary page-card-lg page-card--donate">   
+  <h3 class="title6 donate-title ">Provide In-Kind Software & Services</h3>
```
Comment (roslynwythe) [inline _includes/donate-card.html:2]: Hi @Thinking-Panda This card (under the donation circles graphic) does not match the Figma design.  The card should start with the header ""How" to help us continue to do impactful work" which was present in your initial commit, in donate-card.html, but is now missing. 
image

Comment (Thinking-Panda) [reply _includes/donate-card.html:2]: > Hi @Thinking-Panda there is a problem with overlapping text next to the animated donation gif from screen widths 480 px to 700 px. This is at 600px width. image

Working on it @roslynwythe. I will update soon.

Comment (Thinking-Panda) [reply _includes/donate-card.html:2]: Regarding the card that starts with "How to ....", Bonnie mentioned the change that I have implemented ,in the comments section of the issue #3983

**Comment by Thinking-Panda** [comment]
@roslynwythe , I have fixed the overlapping text and pushed the changes. Let me know about the In Kind donation card information. Which one should I keep "How to..." or "Provide In-KInd..." ?

File: _includes/donate-card.html (hunk 1 above)
Comment (roslynwythe) [reply _includes/donate-card.html:2]: Hi @Thinking-Panda thank you for your prompt explanation.  I apologize for the confusion.  

Please post your progress, availability and ETA for the other requested change.  As I mentioned, this issue is now time-sensitive, so we need to know when we can expect an update.  Thank you!

**Review by roslynwythe — APPROVED** [review]
Awesome work @Thinking-Panda.  The branches have been setup correctly, you described your work and provided screenshots after each update.  Both the Donate and Join Us pages look good in the browser at mobile, desktop and tablet screen sizes.  Thank you for your patience and persistence in this PR.

**Comment by mademarc** [comment]
Review ETA: 4/18/2023
Availability: 7:20 PM

**Review by mademarc — APPROVED** [review]
Hey  @Thinking-Panda the code looks better structured and very readable also  thanks to @roslynwythe in making the marking these changes... well done

**Review by roslynwythe — COMMENTED** [review]
Hi @Thinking-Panda while we're waiting for the final review, I have a minor request.  The second bullet point in your PR is a bit confusing, because the donation-graphic.svg did not actually replace solidarity-pana.svg.  Please correct me if I'm wrong but it appears that solidarity-pana.svg was removed, and donation-graphic.svg was replaced with a more recent design.  Thanks.

**Comment by Thinking-Panda** [comment]
> Hi @Thinking-Panda while we're waiting for the final review, I have a minor request. The second bullet point in your PR is a bit confusing, because the donation-graphic.svg did not actually replace solidarity-pana.svg. Please correct me if I'm wrong but it appears that solidarity-pana.svg was removed, and donation-graphic.svg was replaced with a more recent design. Thanks.

Hi @roslynwythe 
solidarity-pana.svg is the image on the donate page of the published website.
Screenshot 2023-04-19 225008
I have replaced it with the vertical three circle image as per Figma design.
Hope this answers your question.

**Pull request merged**